tests/cases/fourslash/quickInfoJsDocTags1.ts

```diff
@@ -0,0 +1,7 @@
+/// <reference path='fourslash.ts'/>
```

Comment:
The new test is probably not necessary given the number of other tests updated here.